packages/fxa-settings/src/pages/ResetPassword/ResetPassword/index.test.tsx

```diff
@@ -69,13 +69,15 @@ describe('PageResetPassword', () => {
   it('renders a "Remember your password?" link if "canGoBack: true"', () => {
     render(<ResetPassword canGoBack={true} />);
     expect(
-      screen.getByRole('link', { name: 'Remember your password? Sign in' })
+      screen.getByRole('remember-pw-link', {
```

Comment:
This fails for me when I test. It appears that remember-pw-link is the DOM elements id. Maybe just using getByText is good enough?